public/app/plugins/panel/xychart/XYChartTooltip.tsx

```diff
@@ -93,12 +93,14 @@ export const XYChartTooltip = ({ dataIdxs, seriesIdx, data, xySeries, dismiss, i
 
   let footer: ReactNode;
 
-  if (isPinned && seriesIdx != null) {
+  if (seriesIdx != null) {
```

Comment:
@adela-almasan not sure about this change. it's actually expensive to generate links. this means we do it on every mousemove event.

can we avoid doing the link gen unless is pinned or we get get an actual click?